addons/mail/static/src/owl/components/composer/composer.js

```diff
@@ -224,7 +225,7 @@ class Composer extends Component {
 
         const context = {
             // default_parent_id: this.id,
-            default_body: this._textInputRef.comp.getHtmlContent(),
+            default_body: mailUtils.getHtmlContent(this._textInputRef.comp.getContent()),
```

Comment:
With the new logic the full composer should get its default content from the store too I think.

Reply:
Not in the scope of this PR